packages/app/components/Today/QueueCreateButton.tsx

```diff
@@ -51,6 +52,18 @@ export default function TodayPageCreateButton(): ReactElement {
     }
   }
 
+  function onIsOnlineUpdate(e) {
+    setRoomEditDisabled(e.target.checked);
+    if (e.target.checked) {
+      form.setFieldsValue({
+        officeHourName:
+          role === Role.TA
+            ? "Online"
+            : `Online - Professor ${profile.lastName}'s Office Hours`,
```

Comment:
This logic actually needs to be a bit more complicated - we only want to say `Professor X's office hours` if `allow TAs` is unchecked, because that toggle is what makes this queue a Prof queue. so you'll probably need another handler for the `allow TAs` checkbox which:
* Does nothing if we are not online
* sets or removes `Professor X's office hours` if we are online

Also while you're at it you might as well make these arrow functions too :P

Reply:
TBH it might just be easier to set the `value` prop on the room input if we are `Online`, instead of fiddling with the fields through a handler? Then this handler would only set editability of that field and clear the value when its unchecked? I think that would work but not sure